backend/accounts/baseaccount.go

```diff
@@ -165,3 +178,63 @@ func (account *BaseAccount) SetTxNote(txID string, note string) error {
 	account.config.OnEvent(EventStatusChanged)
 	return nil
 }
+
+// ExportCSV implements accounts.Account.
+func (account *BaseAccount) ExportCSV(w io.Writer, transactions []Transaction) error {
+	writer := csv.NewWriter(w)
+	defer writer.Flush()
```

Comment:
A leftover flush. It is now at the very end.

Reply:
uuh good catch, that made the Error() thing useless.